docs/Versioning.md

```diff
@@ -7,18 +7,16 @@ specification, MUST, SHOULD et al are used consistent with [RFC2119](https://too
 
 ONNX defines versioning policy and mechanism for three classes of entities:
 
-* The abstract graph model and the concrete format that serializes it. The graph model and format are versioned atomically and are referred to as the *IR version.* The IR version is  represented by the `ModelProto.ir_version` field.
-* Operators that are referenced by a given ONNX graph. The version of a given operator  is referred to as the *operator version*. The operator version is  represented by the `TBD` field.
-* An ONNX ModelProto that represents a given graph - that is, the contents of a model. We refer to this version as the *model version* and it is represented by the `ModelProto.model_version` field.
+* The abstract model for graphs and operators and the concrete format that represents them. These are always versioned atomically and are referred to as the *IR version.* 
+* Operator specifications that may be referenced by a given ONNX graph. We refer to this as the *operator version*.
```

Comment:
I think it would be more unambiguous to say something like, "The set of operators that may be referenced by a given ONNX graph. We refer to this as the operator set version."

I'd actually like to expunge ALL references to "operator version" from the standard (replacing them with "operator set version"), I think this will help prevent common mistakes from our users.

Reply:
That's not consistent with my understanding of our design from yesterday.